pkgs/development/python-modules/pyeapi/default.nix

```diff
@@ -22,21 +22,44 @@ buildPythonPackage rec {
     sha256 = "13chya6wix5jb82k67gr44bjx35gcdwz80nsvpv0gvzs6shn4d7b";
   };
 
-  propagatedBuildInputs = [ netaddr ];
+  propagatedBuildInputs = [
+    netaddr
+  ];
 
   checkInputs = [
     mock
     pytestCheckHook
   ];
 
-  pytestFlagsArray = [ "test/unit" ];
+  patches = [
+    # Fix usage of collection, https://github.com/arista-eosplus/pyeapi/pull/223
+    (fetchpatch {
+      name = "fix-collection-usage.patch";
+      url = "https://github.com/arista-eosplus/pyeapi/commit/81754f57eb095703cc474f527a0915360af76f68.patch";
+      sha256 = "sha256-ZNBTPRNmXCFVJeRAJxzIHmCOXZiGwU6t4ekSupU3BX8=";
+    })
+  ];
+
+  postPatch = ''
+    # https://github.com/arista-eosplus/pyeapi/pull/223
+    substituteInPlace test/unit/test_utils.py \
+      --replace "collections.Iterable" "collections.abc.Iterable"
+    substituteInPlace pyeapi/api/abstract.py \
+      --replace "from collections" "from collections.abc"
+  '';
```

Comment:
Please fetch https://github.com/arista-eosplus/pyeapi/pull/223/commits/cc9c584e4a3167e3c1624cccb6bc0d9c9bcdbc1c and ingore .github/workflows/ci.yml